src/ray/gcs/client-test.cc

```diff
@@ -0,0 +1,83 @@
+#include "gtest/gtest.h"
```

Comment:
We should really consider adopting the battle-tested Google C++ Style Guide.  It [specifies](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) for the include order:
```
dir2/foo2.h.
C system files.
C++ system files.
Other libraries' .h files.
Your project's .h files.
```

According to this, move hiredis stuff higher than ray headers?